api/src/main/java/org/apache/iceberg/PartitionSpec.java

```diff
@@ -336,17 +338,24 @@ private void checkAndAddPartitionName(String name) {
       checkAndAddPartitionName(name, null);
     }
 
+    Builder checkConflict(boolean check) {
```

Comment:
one more tiny nit, but I think this should be checkConflicts. After that I think we are good to go

Reply:
Done. Thanks!

Reply:
Ok let's just wait for tests now